src/backend/api/services/meals.services.js

```diff
@@ -0,0 +1,23 @@
+const knex = require('../../database');
+
+const selectAllMealsQuery = () => {
+  return knex('meals').select('*');
```

Comment:
It should work without select(*)

Reply:
i will pay attention to that. thanks for your time!